src/game/Entities/Vehicle.cpp

```diff
@@ -197,6 +197,14 @@ VehicleInfo::VehicleInfo(Unit* owner, VehicleEntry const* vehicleEntry, uint32 o
 
 VehicleInfo::~VehicleInfo()
 {
+    std::vector<Unit*> passengers;
```

Comment:
I want to see a stacktrace for this crash if possible to see when the destructor is called and not unboarded yet.